nifi-registry/nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/security/NiFiRegistrySecurityConfig.java

```diff
@@ -101,26 +99,17 @@ protected void configure(HttpSecurity http) throws Exception {
                     .anyRequest().fullyAuthenticated()
                     .and()
                 .exceptionHandling()
-                    .authenticationEntryPoint(http401AuthenticationEntryPoint())
-                    .and()
-                .sessionManagement()
-                    .sessionCreationPolicy(SessionCreationPolicy.STATELESS);
+                    .authenticationEntryPoint(http401AuthenticationEntryPoint());
```

Comment:
This actually looks a little more complex than the authentication entry point I just merged in NiFi. Should the NiFi one contain some of these extra tasks such as logging and setting return content types?

Reply:
Thanks for the note @thenatog. The HTTP Request Logging on NiFi provides HTTP status code details for each URL requested, and Spring Security debug logging could be enabled for additional troubleshooting. With that background, however, it would be worthwhile to consider some of kind of error message string in certain cases. The Authentication Entry Point on NiFi has a few more scenarios to handle, but it sounds worth considering in a follow-on issue.